src/main/java/Duke.java

```diff
@@ -1,10 +1,135 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+import Exception.DeadlineFormatException;
+import Exception.TaskFormatException;
+import Exception.TodoFormatException;
+import Exception.EventFormatException;
+import Exception.TaskAlreadyDoneException;
+import Exception.DoneFormatException;
+import Exception.DeleteFormatException;
+import FileHandler.FileHandler;
+import FileHandler.FileTaskWriter;
+import TaskClass.Deadline;
+import TaskClass.Event;
+import TaskClass.TaskList;
+import TaskClass.ToDo;
+
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    private static final String LOGO = " ____        _\n"
+            + "|  _ \\ _   _| | _____\n"
+            + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+    private static final String LINE_SEPERATOR = "    ____________________________________________________________";
+
+    public static void main(String[] args) throws IOException {
+        System.out.println("    Hello from\n" + LOGO);
+        System.out.println(LINE_SEPERATOR +
+                "\n    Hello! I'm Duke\n" +
+                "    What can I do for you?\n" +
+                LINE_SEPERATOR);
+        Scanner sc = new Scanner(System.in);
+        String sourceFilePath = "data" + File.separator + "task.txt";
+        File source = new File(sourceFilePath);
+        //FileHandler fileHandler;
+        //FileTaskWriter fileTaskWriter;
+        FileTaskWriter fileTaskWriter = new FileTaskWriter(sourceFilePath);
+        try {
+            if(!source.exists()) {
+                if (!source.getParentFile().exists()) {
+                    source.getParentFile().mkdirs();
+                    source.createNewFile();
+                }
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
+            System.out.println(LINE_SEPERATOR + "    Duke can't find the task list.\n" + LINE_SEPERATOR);
+        }
+        FileHandler fileHandler = new FileHandler(source);
+        TaskList tasks = new TaskList(fileHandler.parseToArraylist());
+        //FileWriter sourceToWrite = new FileWriter(sourceFilePath);
+        //fileTaskWriter = new FileTaskWriter(sourceFilePath);
+        String userInput = sc.nextLine();
+        while (!userInput.equals("bye")) {
```

Comment:
Consider breaking down the code implemented in this method into multiple smaller methods for better code organisation and easier debugging, as it's very long